components/script/dom/bindings/proxyhandler.rs

```diff
@@ -166,7 +178,7 @@ pub unsafe fn get_expando_object(obj: HandleObject, expando: MutableHandleObject
 
 /// Get the expando object, or create it if it doesn't exist yet.
 /// Fails on JSAPI failure.
-pub unsafe fn ensure_expando_object(cx: *mut JSContext, obj: HandleObject, expando: MutableHandleObject) {
+pub unsafe fn ensure_expando_object(cx: *mut JSContext, obj: RawHandleObject, mut expando: MutableHandleObject) {
```

Comment:
That was to avoid a Handle::from_raw in proxyhandler.rs.

It's used twice in the codebase. Once in proxyhandler.rs:111, the other time in CodegenRust.py:2593.

Reply:
Because using lifetimed handles would require 2 Handle::from_raw's  (`get_expando_object` and some `extern "C"` function) instead of one into() in the codegen.

Is it ok then or should I switch to `Handle`?

Reply:
This is ok.